docker/Dockerfile.ci_gpu_220816

```diff
@@ -0,0 +1,41 @@
+# CI docker GPU env
```

Comment:
this is used for tensorflow and mxnet? could we keep only one docker for gpu tests?

Reply:
Somehow Tensorflow crashes with CUDA 10.2 and I wasn't able to make it work.

Reply:
all right. let's keep it